repo-config.csv

```diff
@@ -1,6 +1,5 @@
-Repository's Location,Branch,Ignore Glob List,Ignore standalone config
-https://github.com/reposense/testrepo-Alpha.git,master,,
-https://github.com/reposense/testrepo-Beta.git,master,docs**,yes
-https://github.com/reposense/testrepo-Delta.git,master,,
-https://github.com/reposense/RepoSense.git,master,,
-https://github.com/reposense/RepoSense.git,nonExistentBranch,,
+Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config
+https://github.com/reposense/testrepo-Alpha.git,master,,,
+https://github.com/reposense/testrepo-Beta.git,master,java;adocs,docs**,yes
+https://github.com/reposense/testrepo-Delta.git,master,html;css,,
```

Comment:
try to keep it similar to functional as possible. Also, you've accidentally left out the last line.

Reply:
As discussed, this `repo-config.csv` is used for both `netlify` and `user guide` so we should split it.
Keeping it similar to the `functional` copy may be confusing to those who read the `user guide`.

As such, I will make a simpler `sample` `repo-config.csv` under `docs` folder for `user guide` and keep the current one similar to `functional` so we can visualize the output with `netlify`.